### DISCUSSION TITLE
Feedback from the community

### DISCUSSION THREAD

**Discussion by monacofj** [fossguild/coral#137] [Announcements]
### Please, leve your feedback

Tell us your impression about the project.

**Comment by lucasclaros** [comment]
This project was such a cool experience for me! Working on the Snake Game helped me learn a lot about how GitHub works and how to collaborate on open-source projects. It was great to see everyone's contributions come together to build something fun and creative. I really enjoyed the process and appreciate the chance to participate. Thank you for this opportunity!

**Comment by de-abreu** [comment]
Evening, teacher. This project was important for me not to embarrass myself when trying to contribute with Open Source projects. The Git Flow workflow, for example, I found it to be required and used to organize projects much bigger than out little snake game, though with very few changes from what we've experienced here. As such I'll keep this project as a reference for my future endeavors in FOSS development. The only criticism that I have for this project is that I was mislead by some word choices in its documentation that it was to be carried out independently by self-assigned groups of students (something common to other classes in this course) with their own copies of the original repo, instead of a single group encompassing the whole class. As such, I've pushed a Pull Request with some suggestions on how to improve the documentation and possibly solve that issue.

**Comment by Ensuo** [comment]
This project being my first interaction with Open Source as a whole has been very interesting, mostly due to its simplicity and various easy issues I could help solve. With this, I understood the Git Flow workflow better, and also understood how the several ways I can help with the development of a project, varying from solving bugs to simple documentation issues. It has been a learning experience for sure.

**Comment by c-capocasali** [comment]
This project was a great experience. It was also my first time contributing to an Open Source project. I would suggest enhancing the documentation regarding the code itself. The project is well-structured and organized, indeed. However, I believe it would be interesting to include coding conventions and provide more detailed explanations about the project files. For now, this isn't a major issue since the codebase is relatively small, but it could become problematic as the codebase grows in size.

**Comment by shogoshima** [comment]
This project highlighted the incredible potential of open source initiatives. I was impressed by how quickly it evolved, with new features being added and bugs resolved at an impressive pace. Although I’m not new to Git and GitHub, this experience gave me a deeper appreciation for how powerful these tools are. It also underscored the importance of a methodical approach to development, which significantly enhances the efficiency and workflow of a project.

**Comment by ralexg** [comment]
The experience I had with the project was not my first contact with open source development, however, it was the first time where I truly involved myself and felt like my contributions were actually meaningful. I learned a lot about Git, GitFlow and even GitHub itself, and gained confidence with the use of those tools. As Abreu commented above, this project shall be an important reference for future contributions on open source developments.

**Comment by rodrigorcz** [comment]
This project was a pleasant experience, because it helped me learn more about the Open Source Environment. I think the PR creation and review part was the best part of the project, although I am not new to Git, it was the part I had the least contact with before. I also got very interested in the Gitflow environment.

**Comment by JustTheHero** [comment]
This project has been my first significant interaction with Open Source, and it has been an incredibly experience. The simplicity and accessibility of the issues made it easier for me to get involved. Through this experience, I’ve gained a much clearer understanding of the Git Flow workflow and learned how impactful even small contributions can be to a project. It has been exciting to see how collaboration and shared knowledge drive development forward, and I’m grateful for the opportunity to be a part of it. I look forward to contributing to more projects in the future.

**Comment by raultel** [comment]
It was a great experience!
I had the opportunity to learn about git more deeply, to know about opensource licenses, workflows and so much more while having a pleasant and fun experience.

**Comment by Guilherme-Bandeira** [comment]
This project encouraged me to seek to learn and work together at a speed that I had not seen before, it was great to see how it evolved as the days went by and passed through different people, while everyone helped each other throughout the development, It was a new experience for me and one that I enjoyed participating in.

**Comment by furtadojean** [comment]
Through working on this project, I learned a lot about the tools and workflow used in Open Source projects. This will for sure be an invaluable lesson moving forward. Thanks to all other developers who made it possible and helped each other along the way!

**Comment by guisp03** [comment]
This project was a very nice learning experience because it gave me a deeper understanding of how collaboration in open source works. It was quite nice to see everyone contribute and see the project grow further with each update. Also, I liked how, through the Git Flow workflow, the development process was well organized, and through that, I understood a little bit about the important role structured workflows play for larger projects.

**Comment by Gustavo-Ando** [comment]
This project has been an incredible learning opportunity for me as I got better familiarized with Git, Git Flow and GitHub, and although I already had been using these tools, I got to better understand them with a hands-on approach. Furthermore, having the class work together to create what ended up being a somewhat complex game was a fun experience and helped me realize the power of Open Source projects and how quickly they can develop.

**Comment by JoaoVictorDBP** [comment]
Helping with this project was an interesting experience. Even though I wasn't able to help much with the code, I think I was able to learn a lot about how GitHub works and how a community behaves. Thank you for the opportunity!

**Comment by GabrielAkioUrakawa** [comment]
This project was a great introduction to contributing to open source and helped me solidify my understanding of the PR workflow. It was particularly useful as I applied the same process during my first internship this semester. Thank you for the opportunity to learn and grow!

**Comment by joaoprfreitas** [comment]
The project provided a great introduction to Open Source, focusing on learning Git, GitHub, and the Git Flow workflow.
